api/manage.py

```diff
@@ -0,0 +1,92 @@
+"""This script provides basic utility commands for managing the API"""
+
+import os
+import sys
+
+from scripts import dump_test_data, empty_db_tables, load_test_data
+
+
+def install():
+    """Installs all python requirements"""
+    os.system("pip install -r requirements.txt & pip install -r requirements-dev.txt")
+
+
+def run_linter():
+    """Runs linter"""
+    os.system("pylint mod")
+
+
+def run_tests():
+    """Runs pytest and provides coverage information when '--cov' is provided"""
+    if len(sys.argv) == 3:
+        if sys.argv[2] == "--cov":
+            os.system("pytest mod --cov")
+        elif sys.argv[2] == "--cov-xml":
+            os.system("pytest mod --cov --cov-report=xml")
+        else:
+            print("Wrong parameters")
+    else:
+        os.system("pytest mod")
+
+
+def format():
+    """Formats API as specified from GitHub CI"""
+    os.system("black . & isort .")
+
+
+def build():
+    """Builds API as specified from GitHub CI"""
+    os.system("mypy mod & pylint mod & pytest mod --cov --cov-report=xml")
```

Comment:
```suggestion
    os.system("mypy mod & pylint mod & pytest mod --cov")
```
I wouldn't want to generate an xml coverage report whenever I run build
Especially, I think that the xml coverage won't even be .gitignored right now